hvplot/converter.py

```diff
@@ -1097,6 +1097,8 @@ def method_wrapper(ds, x, y):
         style = {}
         if self.datashade:
             operation = datashade
+            if 'cmap' in opts:
+                opts['color_key'] = opts['cmap']
```

Comment:
Won't this overwrite an explicit `color_key` that the user supplied?

Reply:
Forgot to push my other change.